src/bulk-transfer-mapping.ts

```diff
@@ -0,0 +1,71 @@
+import { log } from '@graphprotocol/graph-ts'
+
+import {
+  BatchPrepared as BatchPreparedEvent,
+  ApproveSRRByCommitmentWithProof as ApproveSRRByCommitmentWithProofEvent,
+  MigrateBatch as MigrateBatchEvent,
```

Comment:
remove this event too - it's not used.

Reply:
oh it's on the contract too - let's remove it from BulkTransfer,sol too but don't need to redeploy until after the testing i think. it's just unused code.

Reply:
Okay, I removed it later from contract.